apps/emqx_bridge_greptimedb/test/emqx_bridge_greptimedb_SUITE.erl

```diff
@@ -324,7 +324,7 @@ query_by_clientid(Topic, ClientId, Config) ->
         {"Content-Type", "application/x-www-form-urlencoded"}
     ],
     Body = <<"sql=select * from \"", Topic/binary, "\" where clientid='", ClientId/binary, "'">>,
-    {ok, 200, _Headers, RawBody0} =
+    {ok, _, _Headers, RawBody0} =
```

Comment:
Not sure what's changed, but if the server returns 200+ code, we should maybe verify that.
```suggestion
    {ok, StatusCode, _Headers, RawBody0} =
```

then 
```
?assert(StatusCode >= 200 andalso StatusCode < 300) ?
```

Reply:
Yes, the API results format was changed since 0.6, it would return a non-200 status code when has errors. Let's refactor this checking.

Reply:
@zmstone Refactor in https://github.com/emqx/emqx/pull/12772/commits/64eeeb749483f6bc999a2104b04183bd8c56e491, PTAL